### PR TITLE
fix: remove incorrect margin x and y

### PR DESCRIPTION
**Pull request by ap0nia**
## Summary
I forgot to remove the incorrect `marginX` and `marginY` properties which are overriding the correct `mx` and `my` values.

### CODE REVIEW

File: apps/antalmanac/src/components/Map/Map.tsx [hunk 1]
```diff
@@ -217,7 +217,7 @@ export default function CourseMap() {
         <Box sx={{ width: '100%', display: 'flex', flexDirection: 'column', flexGrow: 1, height: '100%' }}>
             <MapContainer ref={map} center={[33.6459, -117.842717]} zoom={16} style={{ height: '100%' }}>
                 {/* Menu floats above the map. */}
-                <Paper sx={{ zIndex: 400, position: 'relative', my: 2, mx: 6.942, marginX: '15%', marginY: 8 }}>
+                <Paper sx={{ zIndex: 400, position: 'relative', my: 2, mx: 8 }}>
```
Comment (KevinWu098) [inline apps/antalmanac/src/components/Map/Map.tsx]: LGTM to me but the x margin seems a tad low

chrome-capture-2023-9-5

Comment (KevinWu098) [inline apps/antalmanac/src/components/Map/Map.tsx]:
```suggestion
                <Paper sx={{ zIndex: 400, position: 'relative', my: 2, mx: 16 }}>
```

**Review by KevinWu098 — CHANGES_REQUESTED** [review]

File: apps/antalmanac/src/components/Map/Map.tsx (hunk 1 above)
Comment (KevinWu098) [reply apps/antalmanac/src/components/Map/Map.tsx]: Alternatively, keeping the percentage-based MX would probably (?) be fine

Comment (ap0nia) [reply apps/antalmanac/src/components/Map/Map.tsx]: image

Comment (ap0nia) [reply apps/antalmanac/src/components/Map/Map.tsx]: I prefer using rem whenever possible. 1 rem usually translates to 4px depending on the CSS framework's settings (it varies natively).

**Comment by EricPedley** [comment]
leaving this review to Kevin

**Review by KevinWu098 — APPROVED** [review]

**Review by KevinWu098 — APPROVED** [review]
LGTM 🚀

**Pull request merged**